<ui version="4.0" >
 <class>ConversionProgressDialog_base</class>
 <widget class="QDialog" name="ConversionProgressDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Dialog caption...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QTextEdit" name="editTitle" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoFormatting" >
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="html" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Dialog title...&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="textInteractionFlags" >
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelAnimation" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelElapsed" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>10</y>
          <width>82</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text" >
         <string>&lt;h2>Elapsed time:&lt;/h2></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>66</y>
          <width>82</width>
          <height>46</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="labelElapsedSec" >
           <property name="text" >
            <string>0</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSec" >
           <property name="text" >
            <string>sec</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="buttonCancel" >
     <property name="toolTip" >
      <string>Cancel process</string>
     </property>
     <property name="whatsThis" >
      <string>Use this button to cancel the conversion. You will have to manually kill the report creation...</string>
     </property>
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
